src/Item.php

```diff
@@ -474,7 +474,8 @@ public function get_contributors()
                 $name = $this->sanitize($contributor['child'][\SimplePie\SimplePie::NAMESPACE_ATOM_10]['name'][0]['data'], \SimplePie\SimplePie::CONSTRUCT_TEXT);
             }
             if (isset($contributor['child'][\SimplePie\SimplePie::NAMESPACE_ATOM_10]['uri'][0]['data'])) {
```

Comment:
Actually, I wonder if we should not just replace the `isset`s with

```suggestion
            if (($uri = $contributor['child'][\SimplePie\SimplePie::NAMESPACE_ATOM_10]['uri'][0] ?? null) !== null && isset(['data'])) {
```

It is ugly but I think it is still easier to read than the current repeating long chains.

Reply:
this pr looks good @jtojnar, up to you if you want to add more cleanup

Reply:
Yeah, we can do more cleanup separately.